### DISCUSSION TITLE
Website Source Down

### DISCUSSION THREAD

**Discussion by mrasyid2211** [sProDev/kodepos#33] [Q&A] [answered]
Hello soluh, thanks for have made this source code open to public. 

Personally me and my team already using this source code for supporting our private application (especially to gather user address more easier). Recently about 3 days ago, we noticed that the main app source for this code is down until now.  I think i need to tell you about this so you can get a new source and continue improving this code (that actually really powerfull and helpfull for others). Thank you so much Soluh

**Comment by sooluh** [comment]
Hi! Thanks for letting me know about this. It seems like I've run out of compatible websites for this script. Instead, as soon as possible, InshaAllah, I'll switch the method in the existing source code, which previously used scraping (which might burden the website owner), to static data that I'll also provide in this repository. Hopefully, I'll get the most up-to-date data and present it for free to the public.

**Comment by sooluh** [comment, marked as answer]
Hi @mrasyid2211, Alhamdulillah, I just committed to fix this issue. Now, I'm no longer using scraping methods and instead opting for static data. The data, InshaAllah, is the most up-to-date, collected this year (2024). And Biidznillah, the search results are almost identical to those on other websites. Hopefully, it can be helpful.
